WordPress/src/main/java/org/wordpress/android/ui/sitecreation/verticals/NewSiteCreationVerticalsFragment.kt

```diff
@@ -86,10 +74,13 @@ class NewSiteCreationVerticalsFragment : NewSiteCreationBaseFormFragment<NewSite
         fullscreenErrorLayout = rootView.findViewById(R.id.error_layout)
         fullscreenProgressLayout = rootView.findViewById(R.id.progress_layout)
         contentLayout = rootView.findViewById(R.id.content_layout)
+        contentLayout.layoutTransition = LayoutTransition() // animate layout changes
```

Comment:
Nitpick: This was intentionally activated after the header was being hidden for the first time in order to prevent animations when we enter the screen. But since it's not required anymore, I'd suggest putting it into the layout file. Wdyt?

Reply:
Fixed in 93416b63f4c54d3b8db7d5186f6e87f06c516c31.